elected_targets/models.py

```diff
@@ -231,7 +231,7 @@ class Meta:
 
 
 class Statehousemem(Target):
-    target_column = 'legislator_id'
+    target_column = 'district'
```

Comment:
I'm not sure we want to change this.  See how it's used here:
https://github.com/MoveOnOrg/mop/blob/main/signon/models.py#L599-L663

maybe override display